README.md

```diff
@@ -159,17 +217,14 @@ exec($(lab completion xonsh))
 source <(lab completion zsh)
 ```
 
-## What about [GLab](https://github.com/profclems/glab)?
+# Contributing
 
-Both [glab] and `lab` are open-source tools with the same goal of bringing GitLab to your command line and simplifying the developer workflow. In many ways `lab` is to [hub], what [glab] is to [gh].
+See the [contribution guide](CONTRIBUTING.md).
```

Comment:
We welcome all contributors and their contributions to lab!  See the [contribution guide](CONTRIBUTING.md).

@bmeneg, yeah it's silly :) but I do like to encourage others to contribute!

Reply:
:D that's a good point